### DISCUSSION TITLE
React App Deployed on Github Pages

### DISCUSSION THREAD

**Discussion by dzfinance** [community/community#23957] [Other Feature Feedback, Questions, & Ideas]
<p>I deployed my react app using gh-pages. It ran alright for a few month. When I visited today, it only showed me a blank page (this happened before my new commits today attempting to resolve this issue). The page served fine from my local computer. Here are the error messages:<br>
Uncaught SyntaxError: Unexpected token ‘&lt;’ 2.6834ebd1.chunk.js:1<br>
Uncaught SyntaxError: Unexpected token ‘&lt;’ main.a90c1b22.chunk.js:1<br>
Manifest: Line: 1, column: 1, Syntax error.</p>
<p>Your help is greatly appreciated!</p>

**Comment by dzfinance** [comment, marked as answer]
<p>Looks like it’s relative path issue. GH-Pages configures subdirectories to /directory/static/…<br>
Changing the relative paths to /static/… resolves the issue.</p>

**Reply by avii-7** [reply]
Can you explain little bit ? Where should I do changes ? I'm facing same issue.
